src/reuse/lint.py

```diff
@@ -323,11 +329,124 @@ def lint_summary(report: ProjectReport, out=sys.stdout) -> None:
+            )
+        )
+
+    # Missing licenses
+    if report.missing_licenses:
+        help_texts.append(
+            _(
+                "Fix missing licenses: For at least one of the license"
+                " identifiers provided by the 'SPDX-LicenseIdentifier' tags,"
+                " there is no corresponding license text file in the 'LICENSES'"
+                " directory. For SPDX license identifiers, you can simply run"
+                " 'reuse download --all' to get any missing ones. For custom"
+                " licenses (starting with 'LicenseRef-'), you need to add these"
+                " files yourself."
+            )
+        )
+
+    # Unused licenses
+    if report.unused_licenses:
+        help_texts.append(
+            _(
+                "Fix unused licenses: At least one of the license text files in"
+                " 'LICENSES' is not referenced for any file, e.g. by an"
+                " 'SPDX-License-Identifier' tag. Please make sure that you"
+                " either tag the accordingly licensed files properly, or delete"
+                " the unused license text if you are sure that no file or code"
+                " snippet is licensed as such."
+            )
+        )
+
+    # Read errors
+    if report.read_errors:
+        help_texts.append(
+            _(
+                "Fix read errors: At least one of the files in your directory"
+                " cannot be read by the tool. Please check the file"
+                " permissions. You will find the affected files at the top of"
+                " the output as part of the logged error messages."
+            )
+        )
+
+    # Files without copyright and/or licensing information
+    if report.files_without_copyright or report.files_without_licenses:
+        help_texts.append(
+            _(
+                "Fix missing copyright/license information: For one or more"
+                " files, the tool cannot find copyright and/or license"
+                " information. Please add it as a comment in the file header,"
+                " ideally using the 'SPDX-FileCopyrightText' tag. If that's not"
+                " possible, you can use adjacent '.license' files or the"
+                " '.reuse/dep5' file."
+            )
+        )
+
+    # Output help texts in a nicely wrapped format
+    wrapper = TextWrapper(
+        width=80,
+        drop_whitespace=True,
+        break_long_words=False,
+        initial_indent="* ",
+        subsequent_indent="  ",
+    )
+
+    for help_text in help_texts:
+        out.write("\n".join(wrapper.wrap(help_text)))
+        out.write("\n")
+
+
 def add_arguments(parser):
     """Add arguments to parser."""
     parser.add_argument(
         "-q", "--quiet", action="store_true", help=_("prevents output")
     )
+    parser.add_argument(
+        "--no-recommendations",
+        action="store_true",
+        help=_("do not recommend steps to fix found issues"),
+    )
```

Comment:
Do we really need this flag? Sure it is nice to have, but the current output already seems quite verbose so I don't think this flag will do much to reduce the output.

Reply:
I don't have a strong opinion on this. @linozen, would you like to discuss that in a meeting?

Reply:
no strong opinions, either. Let's leave it out then.